Examples/Scripts/Python/common.py

```diff
@@ -41,7 +42,7 @@ def getOpenDataDetector(mdecorator=None):
         if not found:
             msg = (
                 "Unable to find OpenDataDetector factory library. "
-                f"You might need to point {'/'.join(env_vars)} at it"
+                f"You might need to point {'/'.join(env_vars)} to build/thirdparty/OpenDataDetector/factory"
```

Comment:
If you're already changing this, can you add `or other ODD install location`?

Reply:
Sure ! And I will also add it to the doc